exporter/wavefront/example/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"go.opencensus.io/exporter/wavefront"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+
+	"github.com/wavefronthq/wavefront-sdk-go/application"
+	"github.com/wavefronthq/wavefront-sdk-go/histogram"
+	wfsender "github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+var (
+	invCount = stats.Int64("invocation_count", "", stats.UnitDimensionless)
+	workSize = stats.Int64("work_size", "", stats.UnitBytes)
+	wg       = sync.WaitGroup{}
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Configure Wavefront Sender
+	proxyCfg := &wfsender.ProxyConfiguration{
+		Host: "localhost",
+
+		// At least one port should be set below.
+		MetricsPort:      30000, // set this (typically 2878) to send metrics
+		DistributionPort: 40000, // set this (typically 40000) to send distributions
+		TracingPort:      50000, // set this to send tracing spans
+
+		FlushIntervalSeconds: 1, // flush the buffer periodically, defaults to 5 seconds.
+	}
+	sender, _ := wfsender.NewProxySender(proxyCfg)
+
+	appTags := application.New("test-app", "test-service")
```

Comment:
Start a Heartbeat service so the app show up on the "Application Inventory"
https://github.com/wavefrontHQ/wavefront-sdk-go/blob/771cabea2c77cf0db40ea36209e716c8848d29b1/application/heartbeater.go#L26

Reply:
maybe is best to start the Heartbeat on `NewExporter()`